app/derivative-services/raster_resource_derivative_service.rb

```diff
@@ -81,7 +81,7 @@ def instructions_for_display
     {
       input_format: original_file.mime_type.first,
       label: :display_raster,
-      id: resource.id,
+      id: prefixed_id,
       format: "tif",
       srid: "EPSG:3857",
```

Comment:
Using EPSG:3857 for raster resources is fine here?

Reply:
Yeah. It's preferred. The issue is because GeoServer can't autogenerate a shapefile layer in 3857 very well.